ts/src/bingx.ts

```diff
@@ -676,14 +676,47 @@ export default class bingx extends Exchange {
         return this.parseMarkets (markets);
     }
 
+    async fetchInverseSwapMarkets (params) {
+        const response = await this.cswapV1PublicGetMarketContracts (params);
+        //
+        //     {
+        //         "code": 0,
+        //         "msg": "",
+        //         "timestamp": 1720074487610,
+        //         "data": [
+        //             {
+        //                 "symbol": "BNB-USD",
+        //                 "pricePrecision": 2,
+        //                 "minTickSize": "10",
+        //                 "minTradeValue": "10",
+        //                 "minQty": "1.00000000",
+        //                 "status": 1,
+        //                 "timeOnline": 1713175200000
+        //             },
+        //         ]
+        //     }
+        //
+        const markets = this.safeList (response, 'data', []);
+        return this.parseMarkets (markets);
+    }
+
     parseMarket (market: Dict): Market {
         const id = this.safeString (market, 'symbol');
         const symbolParts = id.split ('-');
         const baseId = symbolParts[0];
         const quoteId = symbolParts[1];
         const base = this.safeCurrencyCode (baseId);
         const quote = this.safeCurrencyCode (quoteId);
-        const currency = this.safeString (market, 'currency');
+        let currency = this.safeString (market, 'currency');
+        let checkIsInverse = false;
+        let checkIsLinear = true;
+        const minTickSize = this.safeNumber (market, 'minTickSize');
+        if (minTickSize !== undefined) {
```

Comment:
this check doesn't seem very reliable because in the future `minTickSize` can be added to any market

Reply:
Hmm I'm not sure what a better approach would be, maybe appending a flag to the `fetchInverseSwapMarkets` response that gets parsed?